apps/openssl.cnf

```diff
@@ -60,6 +60,11 @@ default = default_sect
 # included fipsmodule.cnf.
 # fips = fips_sect
 
+# XXX If no providers are activated, the default one is activated implicitly.
+# If you add a section explicitly activating any other providers,
+# you MAY need to explicitly activate the default provider, otherwise
+# some algorithms will become unavailable and system MAY become
+# unreachavble via SSH
```

Comment:
Typo: unreachable.

Also `become unavailable in openssl. As a consequence the system may for example become unreachable via SSH.`

Reply:
Fixed, thanks!